app/src/main/res/layout/screen_patients.xml

```diff
@@ -54,53 +91,31 @@
         app:drawableTint="@color/white100"
         tools:ignore="MissingPrefix,RtlSymmetry" />
 
-      <ImageView
-        android:id="@+id/patients_record_bp_illustration"
+      <org.simple.clinic.widgets.ViewFlipperWithDebugPreview
```

Comment:
Curious, what does `ViewFlipperWithDebugPreview` do?

Reply:
Can we rename `ViewFlipperWithDebugPreview`, something like `SimpleViewFlipper` should do. WDYT?

Reply:
May I suggest `ViewFlipperWithLayoutEditorPreview` instead? This makes it explicit why the class exists, otherwise `SimpleViewFlipper` might indicate that the class has something to do with the domain itself.

Reply:
Also, the fact that this needs to be done us is so indicative of the general quality of the Android UI framework 😥

Reply:
@vinaysshenoy yes, the `Debug` gives a _sense_ that this component should not be part of the production code. Which it is not.